asset/scss-about/_queries.scss

```diff
@@ -0,0 +1,240 @@
+@media screen and (min-width: 769px) {
+  .mobile_version {
+    display: none;
+  }
+
+  .header_text {
+    width: 100vw;
+  }
+}
+
+@media screen and (min-width: 769px) {
+  /* header section */
+  nav,
+  .header_text {
+    padding: 0%;
+  }
+
+  .destop-version {
+    display: block;
+    background-color: #3e3e3e;
+    color: #efefef;
+    margin: 0%;
+    padding: 0%;
+    height: 3vh;
+    width: 100vw;
+  }
+
+  .icon_list {
+    list-style: none;
+    display: flex;
+    justify-content: end;
+    position: absolute;
+    left: 60%;
+    padding: 0%;
+    align-content: flex-end;
+  }
+
+  .icon_list li {
+    padding: 0 1.5em;
+    font-size: xx-large;
+    font-weight: lighter;
+  }
+
+  /* second navigation */
+  .navigation {
+    background-color: white;
+    margin: 0%;
+    display: grid;
+    grid-template-columns: 30% 70%;
+    width: 100vw;
+    font-size: larger;
+    font-weight: 600;
+    justify-content: center;
+    z-index: 1;
+  }
+
+  .right-nav {
+    margin: 0 auto;
+  }
+
+  .main-logo {
```

Comment:
- [ ] Please fix the styling for this logo image on the about page as you managed it on the home page. It's causing a large overflow to the page.

>A screenshot from your project.
![Screenshot5](https://user-images.githubusercontent.com/47625407/124308759-06152f00-db6a-11eb-9d59-9b508bf56903.png)